### PR TITLE
Vanilla SVG implementation

### PR DESCRIPTION
**Pull request by krowter**
Issue [#22](https://github.com/teknologi-umum/decimal-clock/issues/22)

Note: the .svg file contains JavaScript code, so it needs to be viewed in browser or embedded inside html file with `<object>`

### CODE REVIEW

**Comment by herlangga72** [comment]
Minta ss dong hehehe

**Comment by krowter** [comment]
Siap Mas, haha, ini ya

**Review by herlangga72 — APPROVED** [review]
I approve

**Comment by aldy505** [comment]
> Minta ss dong hehehe

@herlangga72 m a n j a

File: vanilla-svg/dist/clock.svg [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+      }
+    }
+
+    function startRotate(selectorId, initial, duration) {
+      const animation = document.getElementById(selectorId);
+      animation.setAttribute('from', `${initial} ${center} ${center}`);
+      animation.setAttribute('to', `${initial + 360} ${center} ${center}`);
+      animation.setAttribute('dur', duration);
+
+      animation.beginElement();
```
Comment (ronnygunawan) [inline vanilla-svg/dist/clock.svg:133]: Jangan pake animation kaya di README.md. Nanti ketika ada skipped frame jamnya jadi ngga akurat.

**Review by ronnygunawan — CHANGES_REQUESTED** [review]

**Comment by herlangga72** [comment]
> > Minta ss dong hehehe
> 
> @herlangga72 m a n j a

Gw sakit beneran. Seharian diatas kasur.  Sakit panas... Ealah kenapa begini dah

File: vanilla-svg/dist/clock.svg (hunk 1 above)
Comment (krowter) [reply vanilla-svg/dist/clock.svg]: Wah baru tahu soal masalah skipped frame ini. 
Sip, sudah saya ganti pakai `requestAnimationFrame`

**Review by ronnygunawan — APPROVED** [review]

**Pull request merged**